system-variables.md

```diff
@@ -4149,12 +4149,16 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
 
 ### `tidb_scatter_region`
 
-- 作用域：GLOBAL
+- 作用域：SESSION | GLOBAL
```

Comment:
keep the original global level?

Reply:
Yes，Current implement should add session level sys var change and also keep global level implementation.